cmd/api/src/api/error.go

```diff
@@ -114,10 +118,15 @@ func BuildErrorResponse(httpStatus int, message string, request *http.Request) *
 // HandleDatabaseError writes an error (not found or other) depending on the database error encountered
 // Alternate: FormatDatabaseError()
 func HandleDatabaseError(request *http.Request, response http.ResponseWriter, err error) {
+	log.Infof("Handling database error: %v", err)
```

Comment:
I'm not sure we want to log this as `Info` as it could get quite noisy

Reply:
That was for debugging - I will remove it!